tools/validators/instance_validator/validate/telemetry_error.py

```diff
@@ -39,8 +39,8 @@ def __eq__(self, other):
   def GetPrintableMessage(self):
     """Returns a human-readable message that explains this error."""
 
-    msg = "Error for entity [" + self.entity + "]"
-    if self.point:
-      msg += ", point [" + self.point + "]"
-    msg += ": " + self.message
+    msg = "- entity [" + self.entity + "]"
+    if str(self.point):
+      msg += ", point [" + str(self.point) + "]"
```

Comment:
self.point should always be a string or None according to the docstring, so str() isn't necessary here.

I like string.format for this kind of thing though. (Sorry, I was still de-rusting my Python when I wrote this code.)

Reply:
it doesn't work without the str, the error message is trying to cast Yaml to str